addons/account/wizard/account_payment_register.py

```diff
@@ -255,6 +253,8 @@ def _get_batches(self):
 
         if len(lines.company_id.root_id) > 1:
             raise UserError(_("You can't create payments for entries belonging to different companies."))
+        if len(lines.company_id.filtered(lambda c: c.root_id not in lines.company_id)) > 1:
```

Comment:
```suggestion
        if len(lines.company_id.root_id) > 1:
```

Reply:
We already check
`if len(lines.company_id.root_id) > 1` 
to avoid registering payments from different companies
With
`if len(lines.company_id.filtered(lambda c: c.root_id not in lines.company_id))` we ensure that siblings are accompanied by a parent